src/config/ConfigManager.cpp

```diff
@@ -683,6 +683,12 @@ std::string CConfigManager::getMainConfigPath() {
     if (!g_pCompositor->explicitConfigPath.empty())
         return g_pCompositor->explicitConfigPath;
 
+    if (const char *cfg = getenv("HYPRLAND_CONFIG"); cfg != NULL) {
```

Comment:
no {}

no `!= NULL`

const are CAPS

you can use auto

```
if (const auto CFG_ENV = getenv("HYPRLAND_CONFIG"); CFG_ENV)
```

Reply:
should be resolved now

Reply:
should actually be resolved now